sdl2_mixer/VITABUILD

```diff
@@ -1,37 +1,43 @@
 pkgname=sdl2_mixer
-pkgver=2.6.3
-pkgrel=3
+pkgver=2.7.0
+pkgrel=1
+gitrev=eb005518b0085244630ccfb41977f818af45d2ea
```

Comment:
related version thing. why do not we just use the `SDL2` branch if we have to follow the upstream branch?

Reply:
Because it's tested working commit. Later commits break cmake again. I have no time to test every commit there is.

Reply:
ok. i got it. just change the `pkgver` value to `2.6.99`, `2.7.0-dev`, `2.7.0-pre` or whatever else.

Reply:
done

Reply:
well, looks like we can't have `-` in pkgname. changed to 2.6.99